torch/csrc/distributed/rpc/testing/faulty_process_group_agent.cpp

```diff
@@ -14,14 +14,16 @@ FaultyProcessGroupAgent::FaultyProcessGroupAgent(
     int numSendRecvThreads,
     std::chrono::milliseconds rpcTimeout,
     const std::vector<std::string>& messagesToFail,
+    std::unordered_map<std::string, float> messagesToDelay,
     int failNumSends)
     : ProcessGroupAgent(
           std::move(workerName),
           std::move(pg),
           numSendRecvThreads,
           rpcTimeout),
       failNumSends_(failNumSends),
-      messageTypesToFail_(parseMessagesToFailInput(messagesToFail)) {}
+      messageTypesToFail_(parseMessagesToFailInput(messagesToFail)),
+      messagesToDelay_(parseMessagesToDelay(messagesToDelay)) {}
```

Comment:
rename this to `messageTypesToDelay`? (just for consistency with `messageTypesToFail`)